org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -617,55 +619,131 @@ public void moveProcessButtonsToBeforeRight() {
 	 */
 	protected void processQueryValue() {
 		isQueryByUser = true;
+		boolean splitValue = false;
+		
+		if (isAutoComplete) {
+			testQueryForAutoComplete();
+		}else {
+			String separator = MSysConfig.getValue(MSysConfig.IDENTIFIER_SEPARATOR, "_", Env.getAD_Client_ID(Env.getCtx()));
+			String[] values = queryValue.split("[" + separator.trim()+"]");
+
+			if (values.length > 1) {
+				splitValue = true;
+				testQueryForSplit(values);
+			} 
+
+			if (m_count <= 0) {
+				testQueryForEachIdentify();
+			}
+		}
+		
+		if (m_count > 0) {
+			executeQuery();
+			renderItems();
+		} else if (!splitValue) {
+			editors.get(0).setValue(queryValue);
+		}
+		isQueryByUser = false;
+	}
+
+	protected void testQueryForAutoComplete() {
+		WEditor autocompleteEditor = null;
+		
+		if (Util.isEmpty(autoCompleteSearchColumn) && identifiers.size() > 0) {
+			autocompleteEditor = identifiers.get(0);
+		}else if (!Util.isEmpty(autoCompleteSearchColumn) && identifiers.size() > 0) {
+			autocompleteEditor = identifiers.stream()
+					.filter(editor -> editor.getColumnName().equals(autoCompleteSearchColumn))
+					.findFirst().orElse(null);
+		}
+		
+		
+		if (autocompleteEditor != null) {
+			try{
+				autocompleteEditor.setValue(queryValue);
+				testCount(false);
+			}catch(Exception ex){
+				// => don't run test in case not success set value
+				log.log(Level.SEVERE, "error", ex.getCause());
+			}
+		}else {
+			// => don't run test in case not found auto complete column
+			log.log(Level.SEVERE, String.format("can't found column for autocomplete query for field %s - field id %s. first identify column on m_table need to exists on identifies of info window", 
+					m_gridfield.getColumnName(), m_gridfield.getAD_Column_ID()));
```

Comment:
autoCompleteSearchColumn not empty - "Auto complete search column (%s) not found for field %s (field id %s). ", autoCompleteSearchColumn, m_gridfield.getColumnName(), m_gridfield.getAD_Column_ID()

autoCompleteSearchColumn empty - "Info window has no identifier columns"